File: Module/api/src/main/java/com/example/TestController.java
package com.example;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * created by Gyunny 2022/02/13
 */
@RestController
public class TestController {

    @GetMapping
    public String test() {
        return "Test";
    }

}

File: Spring_Swagger/src/main/kotlin/com/swagger/controller/TestController.kt
package com.swagger.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/**
 * created by Gyunny 2021/10/23
 */
@RestController
class TestController {

    @GetMapping
    fun test(): String {
        return "test"
    }

}